docs/performance/tune-performance.md

```diff
@@ -245,17 +242,17 @@ sess_options.graph_optimization_level = rt.GraphOptimizationLevel.ORT_ENABLE_ALL
 ```
 
 * Thread Count
-  * `sess_options.intra_op_num_threads = 2` controls the number of threads to use to run the model
+  * `sess_options.intra_op_num_threads = 2` controls the number of threads to use to run the model.
 * Sequential vs Parallel Execution
-  * `sess_options.execution_mode = rt.ExecutionMode.ORT_SEQUENTIAL` controls whether the operators in the graph run sequentially or in parallel. Usually when a model has many branches, setting this option to false will provide better performance.
+  * `sess_options.execution_mode = rt.ExecutionMode.ORT_SEQUENTIAL` controls whether the operators in the graph run sequentially or in parallel. Usually when a model has many branches, setting this option to `ORT_PARALLEL` will provide better performance.
```

Comment:
I was not 100% on this one but I think "false" refers to the "ORT_PARALLEL" option.